packages/fx-core/tests/question/create.test.ts

```diff
@@ -1259,7 +1259,7 @@ describe("scaffold question", () => {
           } else if (question.name === QuestionNames.CustomCopilotRag) {
             const select = question as SingleSelectQuestion;
             const options = await select.dynamicOptions!(inputs);
-            assert.isTrue(options.length === 2);
+            assert.isTrue(options.length === 3);
             return ok({ type: "success", result: CustomCopilotRagOptions.customize().id });
```

Comment:
minor: would you mind help change `CustomCopilotRagOptions.customize().id` to `CustomCopilotRagOptions.azureAISearch().id` according to the test case name.

Reply:
maybe we could also raise a pr for all these test cases refractoring later.